### PR TITLE
feat: Support non-ordered `LIMIT` and `OFFSET`

### LINKED ISSUE #72: Support `OFFSET LIMIT` syntax for `PostgreSQL`
Currently if we will write `select * from some_table offset 10 limit 10` we will get syntax error, but actually all fine

### PR DESCRIPTION
**Pull request by viladimiru**
Closes #72

### CODE REVIEW

File: src/parsing/parsers/postgresql/jison/select/offset_clause.test.json [hunk 1]
```diff
@@ -38,5 +48,23 @@
     "expectedResult": {
       "lowerCase": false
     }
+  },
+  {
+    "namePrefix": "should suggest OFFSET",
+    "beforeCursor": "SELECT * FROM testTable WHERE column = 1 ",
+    "afterCursor": "",
+    "noErrors": true,
+    "containsKeywords": ["OFFSET"],
```
Comment (NikitaShkaruba) [inline src/parsing/parsers/postgresql/jison/select/offset_clause.test.json:57]: Do we want to add `LIMIT` to keywords here?

File: src/parsing/parsers/postgresql/jison/select/select_conditions.jison [hunk 2]
```diff
@@ -146,14 +145,53 @@ OptionalSelectConditions_EDIT
...
+
+LimitOffsetClause_EDIT
+ : LimitClause_EDIT OptionalOffsetClause
+ | LimitClause OffsetClause_EDIT
+ | OffsetClause_EDIT OptionalLimitClause
+ | OffsetClause LimitClause_EDIT
+ ;
+
+
+
```
Comment (NikitaShkaruba) [inline src/parsing/parsers/postgresql/jison/select/select_conditions.jison:190]: There are two excess newlines here

**Review by NikitaShkaruba — APPROVED** [review]
Great job!

File: src/parsing/parsers/postgresql/jison/select/offset_clause.test.json (hunk 1 above)
Comment (viladimiru) [reply src/parsing/parsers/postgresql/jison/select/offset_clause.test.json]: It's will not be excess for tests 👍

**Review by NikitaShkaruba — APPROVED** [review]

**Pull request merged**